en_US.ftl

```diff
@@ -56,6 +59,7 @@ length-of-sections = Length of sections
 length-of-sections-tooltip = This refines the results. Think if it's like goldilocks. If the number is too big it will miss really good highlights, and if the number is too small it might produce more false positives. Each streamer is different so we are giving you the option to experiment. The default should be fine. Recommended: ${VODinatorInitial.lenSection}
 
 compare-sections = Compare sections
+# XXX: TBH, I don't really understand what the last part means in english either ("This determines how relative you want your edits to be.")
 compare-sections-tooltip = This refines the results. Each section is compared to its neighboring sections to determine whether or not data is an outlier. Every bit of data is relative since there is so much fluctuation in audience behavior and retention. If you select 1 it will look at 1 section in both directions. If you select 5 it will look 5 sections in each direction. This determines how relative you want your edits to be. The default should be fine. Recommended: {$VODinitialCompareSection}
```

Comment:
This is bloody hilarious, that whole paragraph makes no sense at all - as said, copy editing is likely in order :)